pkg/graph/schema.resolvers.go

```diff
@@ -629,6 +629,92 @@ func (r *cedarRoleTypeResolver) Description(ctx context.Context, obj *models.Ced
 	return obj.Description.Ptr(), nil
 }
 
+// AiSolnCatgOther is the resolver for the aiSolnCatgOther field.
+func (r *cedarSoftwareProductsResolver) AiSolnCatgOther(ctx context.Context, obj *models.CedarSoftwareProducts) (*string, error) {
+	return obj.AISolnCatgOther.Ptr(), nil
+}
+
+// APIDescPubLocation is the resolver for the apiDescPubLocation field.
+func (r *cedarSoftwareProductsResolver) APIDescPubLocation(ctx context.Context, obj *models.CedarSoftwareProducts) (*string, error) {
+	return obj.APIDescPubLocation.Ptr(), nil
+}
+
+// APIDescPublished is the resolver for the apiDescPublished field.
+func (r *cedarSoftwareProductsResolver) APIDescPublished(ctx context.Context, obj *models.CedarSoftwareProducts) (*string, error) {
+	return obj.APIDescPublished.Ptr(), nil
+}
+
+// APIFHIRUse is the resolver for the apiFHIRUse field.
+func (r *cedarSoftwareProductsResolver) APIFHIRUse(ctx context.Context, obj *models.CedarSoftwareProducts) (*string, error) {
+	return obj.APIFHIRUse.Ptr(), nil
+}
+
+// APIFHIRUseOther is the resolver for the apiFHIRUseOther field.
+func (r *cedarSoftwareProductsResolver) APIFHIRUseOther(ctx context.Context, obj *models.CedarSoftwareProducts) (*string, error) {
+	return obj.APIFHIRUseOther.Ptr(), nil
+}
+
+// APIHasPortal is the resolver for the apiHasPortal field.
+func (r *cedarSoftwareProductsResolver) APIHasPortal(ctx context.Context, obj *models.CedarSoftwareProducts) (*bool, error) {
+	return obj.APIHasPortal.Ptr(), nil
+}
+
+// ApisAccessibility is the resolver for the apisAccessibility field.
+func (r *cedarSoftwareProductsResolver) ApisAccessibility(ctx context.Context, obj *models.CedarSoftwareProducts) (*string, error) {
+	return obj.ApisAccessibility.Ptr(), nil
+}
+
+// ApisDeveloped is the resolver for the apisDeveloped field.
+func (r *cedarSoftwareProductsResolver) ApisDeveloped(ctx context.Context, obj *models.CedarSoftwareProducts) (*string, error) {
+	return obj.ApisDeveloped.Ptr(), nil
+}
+
+// DevelopmentStage is the resolver for the developmentStage field.
+func (r *cedarSoftwareProductsResolver) DevelopmentStage(ctx context.Context, obj *models.CedarSoftwareProducts) (*string, error) {
+	return obj.DevelopmentStage.Ptr(), nil
+}
+
+// SoftwareProducts is the resolver for the softwareProducts field.
+func (r *cedarSoftwareProductsResolver) SoftwareProducts(ctx context.Context, obj *models.CedarSoftwareProducts) ([]*model.CedarSoftwareProductItem, error) {
+	softwareProducts := obj.SoftwareProducts
+
+	if len(softwareProducts) == 0 {
+		return nil, nil
+	}
+
+	var softwareProductItems []*model.CedarSoftwareProductItem
+	for _, softwareProduct := range softwareProducts {
+		softwareProductItem := &model.CedarSoftwareProductItem{
+			APIGatewayUse:                  softwareProduct.APIGatewayUse.Ptr(),
```

Comment:
Calling `.Ptr` on a `zero.BoolFrom` returns `nil`, so referring to my other comment, we should probably use `null` for the boolean values.